README.md

```diff
@@ -140,7 +140,7 @@ This repository uses [git submodules](https://git-scm.com/book/en/v2/Git-Tools-S
 git submodule update --init --recursive
 ```
 
-OpenConsole.sln may be built from within Visual Studio or from the command-line using MSBuild. To build from the command line:
+OpenConsole.sln may be built from within Visual Studio or from the command-line using MSBuild. To build from the command line (`cmd.exe` should be used):
```

Comment:
```suggestion
OpenConsole.sln may be built from within Visual Studio or from the command-line using MSBuild. To build from the command line, find your shell below.

### PowerShell

```powershell
Import-Module .\tools\OpenConsole.psm1
Set-MsBuildDevEnvironment
Invoke-OpenConsoleBuild
\`\`\`

### CMD
```

Minor problem. I wanted to fix all the markdown in the editor, but I can't.

This:

```
\`\`\`
```

should definitely just be ` ``` `... but there's limits to GitHub's expressiveness ;)

Reply:
I figured we should just go ahead and make this comprehensive so as to reduce overall changes to this file :)

Reply:
Done :)